src/navigation/wallet/screens/CreateMultisig.tsx

```diff
@@ -546,7 +541,7 @@ const CreateMultisig = () => {
                     });
                   }}>
                   <Column>
-                    <OptionTitle>Segwit</OptionTitle>
+                    <OptionTitle>{t('Segwit')}</OptionTitle>
```

Comment:
Are we sure we want to translate this ?

Reply:
Not sure, seemed like a missed strings. I can remove

Reply:
Yes, thanks !